src/config/dynamic-theme-fixes.config

```diff
@@ -13694,14 +13696,27 @@ img[alt="People using Notion"]
 .api-section-image .next-image
 .visual .next-image
 .logo svg > :first-child
+img[alt="Books on a shelf"]
+img[alt ="Some building blocks"]
+img[alt ="A team of people collaborating"]
+.logo-limit-height
+.notranslate::after
 
 CSS
 body {
-    background-color: var(--darkreader-neutral-background) !important;
+background-color: var(--darkreader-neutral-background) !important;
 }
 .notion-divider-block div div {
-    border-bottom: 1px solid ${rgba(55, 53, 47, 0.4)} !important;
-}
+border-bottom: 1px solid ${rgba(55, 53, 47, 0.4)} !important;}
+[data-darkreader-inline-fill]
+{fill: #ffffff !important;}
```

Comment:
Don't try to specify general dark reader selectors, use a selector that target the element instead.

Reply:
I think that was the code that was already written before me. Do you want me to remove them or see a better fix?